tests/models/embedding/language/test_embedding.py

```diff
@@ -43,3 +53,9 @@ def test_models(
         name_1="vllm",
         tol=1e-2,
     )
+
+    if "VLLM_ATTENTION_BACKEND" in os.environ:
+        if prior_attn_backend_env is None:
+            del os.environ["VLLM_ATTENTION_BACKEND"]
+        else:
+            os.environ["VLLM_ATTENTION_BACKEND"] = prior_attn_backend_env
```

Comment:
To avoid missing this when the test fails (which may be relevant when multiple tests are run in sequence), I recommend placing this in a pytest fixture.